src/pages/overview/pdf-services-api/howtos/create-pdf.md

```diff
@@ -426,58 +438,61 @@ Please refer the [API usage guide](../api-usage.md) to understand how to use our
 ```javascript 
 // Get the samples from https://www.adobe.com/go/pdftoolsapi_java_samples
 // Run the sample:
-// mvn -f pom.xml exec:java -Dexec.mainClass=com.adobe.pdfservices.operation.samples.createpdf.CreatePDFFromStaticHTML
+// mvn -f pom.xml exec:java -Dexec.mainClass=com.adobe.pdfservices.operation.samples.htmltopdf.StaticHTMLToPDF
 
-public class CreatePDFFromStaticHTML {
+public class StaticHTMLToPDF {
```

Comment:
As discussed @anubhav-mhjn, will be keeping the htmltopdf under create PDF because only Java SDK will be changed in this PR.

CC: @adityaganotra @Prerit12